src/usr/local/php/unraid-tailscale-utils/unraid-tailscale-utils/Info.php

```
@@ -405,4 +424,51 @@ public function connectedViaTS(): bool
     }
+
+    /**
+     * @return array<int>
+     */
+    public function getAllowedFunnelPorts(): array
+    {
+        $allowedPorts = array();
+
+        if (isset($this->status->Self->CapMap)) {
+            $prefix = "https://tailscale.com/cap/funnel-ports?ports=";
+            foreach ($this->status->Self->CapMap as $cap => $value) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$value'. (Unused Code Rules)

(UnusedLocalVariable)